jupyterhub/apihandlers/groups.py

```diff
@@ -34,7 +34,7 @@ def find_group(self, group_name):
 
 
 class GroupListAPIHandler(_GroupAPIHandler):
-    @needs_scope('read:groups')
+    @needs_scope('read:groups', 'read:groups:names', 'read:groups:roles')
```

Comment:
```suggestion
    @needs_scope('read:groups', 'read:groups:name', 'read:groups:roles')
```

Reply:
Good catch, @IvanaH8!

@0mar the fact that tests pass with this bug suggests we have some missing test coverage for the `read:group:name` scope.

Reply:
Thanks! I'll fix it and add some tests.